api/src/commonMain/kotlin/Model.kt

```diff
@@ -80,6 +80,11 @@ data class ParentBuildDto(
     val version: String
 )
 
+@Serializable
+data class BaselineDto(
+    val version: String
+)
```

Comment:
maybe it should union ParentBuildDto & BaselineDto to Dto, for example:

> @Serializable
> data class BuildDto(
>     val version: String
> )

Reply:
BuildVersionDto